app/poros/npc.rb

```diff
@@ -1,11 +1,29 @@
+require 'csv'
+
 class NPC
   attr_reader :name,
+              :gender,
               :ancestry,
-              :class
+              :background,
+              :speed,
+              :class,
+              :stats
+
+  def initialize(ancestry, class_data)
+    @name = "#{Faker::Games::WarhammerFantasy.hero.split(' ')[0]} #{Faker::Science.scientist.split(' ')[0]} #{Faker::FunnyName.two_word_name.split(' ')[1]}"
+    @gender = File.read('app/assets/data/genders.txt').split("\n").sample
+    @ancestry = ancestry[:name]
+    @background = create_npc_background
+    @class = class_data[:name]
+    @speed = ancestry[:speed][:walk]
+    @stats = Stats.new(ancestry, @background, class_data)
```

Comment:
This is where things start to get complex.  This Stats class is primarily made up of 3 other classes.  

Likely when I expand and do the middle row and right row, these will be equally complex